### ISSUE TITLE
Hosting parseproject_deploy.jar as part of a release

### ISSUE THREAD

**Issue by jin** [johnynek/bazel-deps#160]
Some projects are depending on https://github.com/vmax/bazel-deps-deploy, which is the uber/deploy jar of parseproject. Is it possible to build the deploy jar and host it directly on this repo's releases?

**Comment by johnynek** [comment]
Yeah, I am willing to do that. The trouble is when should we publish it. Maybe a timed release?

What do you think?

**Comment by jin** [comment]
I think a simple monthly release is good enough for now.

**Comment by vmax** [comment]
`vmax/bazel-deps-deploy` has been moved to [`oferb/startupos_binaries`](https://github.com/oferb/startupos-binaries)

**Comment by igorgatis** [comment]
I think adding a release binary would help adoption of this tool. Bazel by itself has a somewhat steep learning curve. It is not obvious for the newcomers they don't need to make bazel-deps *part of* their own bazel project. All they need is a .jar file which could be host here.

**Comment by vmax** [comment]
@johnynek I'll be happy to setup a CircleCI pipeline which will deploy whatever is merged to `master` to Github Releases page. Are you interested?

**Comment by johnynek** [comment]
I’m very interested. This would be highly appreciated.

**Comment by haikalpribadi** [comment]
Hi @johnynek , have you got a chance to review PR #247 from @vmax? It should resolve this issue, which would be really nice. :)

Thanks!

**Comment by johnynek** [comment]
I let it slip then saw a red PR and overlooked his point about that being expected.

Sorry for the delay.

**Comment by haikalpribadi** [comment]
Thanks, @johnynek ! We look forward to seeing the auto-release working! :)

**Comment by vmax** [comment]
@johnynek friendly ping to remind about my PR 🙂
